README.md

```diff
@@ -29,6 +29,25 @@ NuGet Packages
 
 To make it easer for you to develop with the Google APIs using the Google API client Library for .NET we have released a number of NuGet packages. A full list of all the packages avaliable can be found at [Google.APIs nuget](https://www.nuget.org/packages?q=google.apis&prerelease=true&sortOrder=relevance)
 
+Source for APIs
+=================================
+
+The source code for the individual Google APIs is programmatically generated using the [Discovery API](https://developers.google.com/discovery/).   Using the following formula, you can find the source for any of the API .dlls.
+
+Replace APIName wth the name of the API.
+Replace APIVERSION with the version of the api.
+
+https://developers.google.com/resources/api-libraries/download/APINAME/APIVERSION/csharp
+
+Note: The name of the API used in the Discovery API is not always the same as the common name the API is refered to. For instance we normaly refer to drive as the Google Drive API but its just drive, same with Google Analytics. 
```

Comment:
s/normaly/normally/

Are you sure the Note here is actually useful. It seems obvious that the API identifier can be different from its commonly used name.